packages/yew-macro/tests/html_macro/component-fail.stderr

```diff
@@ -419,6 +410,15 @@ error[E0599]: no method named `children` found for struct `ChildPropertiesBuilde
    |
    = note: this error originates in the macro `html` (in Nightly builds, run with -Z macro-backtrace for more info)
 
+error[E0609]: no field `children` on type `ChildProperties`
+  --> tests/html_macro/component-fail.rs:94:10
+   |
+94 |         <Child ..ChildProperties { string: "hello".to_owned(), int: 5 }>
+   |          ^^^^^ unknown field
```

Comment:
This is a bad span. Can we do better?

Reply:
I agree in principle, but don't want to block this PR with this. The errors were like this before, even before #2729 . This PR only affects the order in which errors are reported, since they emit from a non-const context again. Don't ask me why that changes the order of error reporting though, idk :)